src/chrome/content/rules/headline24.de.xml

```diff
@@ -0,0 +1,18 @@
+<ruleset name="Headline24.de">
```

Comment:
This should be spited to a separate branch/PR. Please also add hosts from `*.headline24.de`.